src/NHibernate/Impl/FilterImpl.cs

```diff
@@ -102,12 +102,12 @@ public object GetParameter(string name)
 		}
 
 		/// <summary>
-		/// Get a span of collection parameter by name. <see langword="null" /> if the parameter is not a collectino or
-		/// there is no such parameter exist.
+		/// Get the span of a value list parameter by name. <see langword="null" /> if the parameter is not a value list
+		/// or if there is no such parameter.
```

Comment:
I have changed "collection" for "value list" for consistency with the xml comment of `SetParameterList` above.